integreat_cms/cms/templates/pois/poi_form_sidebar/related_contacts_box.html

```diff
@@ -0,0 +1,28 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    message-square
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% trans "Related contacts" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    {% with poi_form.instance.contacts.all as contacts %}
+        <div>
+            <div class="help-text">
+                {% if contacts %}
+                    {% trans "This location is currently referred to in those contacts." %}
+                {% else %}
+                    {% trans "This location is not currently referred to in any contact." %}
+                {% endif %}
+            </div>
+            {% for contact in contacts %}
+                <a href="{% url 'edit_contact' contact_id=contact.id region_slug=request.region.slug %}"
+                   class="block pt-2 hover:underline">
+                    <i icon-name="pen-square" class="mr-2"></i> {{ contact.title }} {{ contact.name }}
```

Comment:
```suggestion
                    <i icon-name="pen-square" class="mr-2"></i> {{ contact }}
```
I think we should use the the `__str__` method for contacts as it is currently being implemented in #3142, because we decided to have some more complex logic how to display a contact